nav2_mppi_controller/include/nav2_mppi_controller/motion_models.hpp

```diff
@@ -71,42 +63,33 @@ class MotionModel
    */
   virtual void predict(models::State & state)
   {
-    // Previously completed via tensor views, but found to be 10x slower
```

Comment:
Curious - did you try this and was it better in Eigen vs xtensor? 

Also, please readd, I left it commented out for the reason of having a reference for later use

Reply:
I removed it because the xtensor implementation was copying shifted tensor, which is not the case now after the introduction of acceleration constraints. Should keep it removed? Or add it back.